sevntu-checks/.ci/eclipse-compiler-javac.sh

```diff
@@ -12,8 +12,8 @@ if [ -z "$2" ]; then
     exit 1
 fi
 
-ECJ_JAR="ecj-4.11.jar"
-ECJ_MAVEN_VERSION="R-4.11-201903070500"
+ECJ_JAR="ecj-4.14.jar"
+ECJ_MAVEN_VERSION="R-4.14-201912100610"
```

Comment:
Main project is on 4.11 .

Reply:
There is some problem with 4.11 download, see comments above.
I tried to relauch Travis for main project for this to reproduce problem - but it works fine in main project. So I am not sure what is wrong.
I am ok with different version, if it works. It should not block PR as it is unrelated issue.

Reply:
@rnveach , please take a look - https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/788#issuecomment-568599482

Reply:
I don't have a problem with the new version, as it is just a comment. I was pointing out this is different than main project and we are still bringing in main project's eclipse settings so I wasn't sure if there would be some conflict. As long as the setting file is backwards compatible it should be fine I guess, but we should look into upgrading main project.